internal/testing/e2e/e2e

```diff
@@ -148,8 +148,6 @@ for command in "${ingestion_commands[@]}"; do
   echo @@@@ Waiting
   sleep 60
 
-  go run ${GUAC_DIR}"/cmd/guacone" certifier osv -p=false
```

Comment:
removed `guacone certifier osv` as `--add-vuln-on-ingest=true` is already being called. This caused duplicate vulns to show up and break the e2e test.